step.yml

```diff
@@ -136,7 +136,15 @@ inputs:
       description: |-
         Specify the output pattern with golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)
       is_required: true
-      is_expand: false                    
+      is_expand: false
+  - permanent_download_url_map_format: "{{range $index, $element := .}}{{if $index}}|{{end}}{{$element.File}}=>{{$element.URL}}{{end}}"
+    opts:
+      title: Format for the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output
+      summary: You can change the format of the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output
```

Comment:
I'd rephrase it to something like this: You can customize the format of the `BITRISE_PERMANENT_DOWNLOAD_URL_MAP` output so that it fits the next Step you wish to use the output with.